index-test.js

```diff
@@ -449,6 +453,12 @@ describe(`reactElementToJSXString(ReactElement)`, () => {
     ).toEqual(`<TESTCOMPONENT />`);
   });
 
+  it('reactElementToJSXString(<DefaultPropsComponen />)', () => {
+    expect(
+      reactElementToJSXString(<DefaultPropsComponent />)
+    ).toEqual(`<DefaultPropsComponent />`);
```

Comment:
The point is that if a passed property is equivalent to the default it should not be shown. Maybe it should be an option than the default behavior. `<DefaultPropsComponent />` and `<DefaultPropsComponent test="test" />` are equivalent but the latter is showing props that are inferred from the defaults.